exonum/src/runtime/mod.rs

```diff
@@ -138,24 +210,39 @@ impl<'a> Default for ArtifactInfo<'a> {
     }
 }
 
+/// An accessory structure that aggregates root objects hashes of runtime service
+/// information schemas with the root hash of runtime information schema itself.
 #[derive(Debug, PartialEq, Default)]
 pub struct StateHashAggregator {
+    /// List of hashes of the root objects of runtime information schemas.
     pub runtime: Vec<Hash>,
+    /// List of hashes of the root objects of service instances schemas.
     pub instances: Vec<(ServiceInstanceId, Vec<Hash>)>,
```

Comment:
Would Map/SortedMap be more convenient?

Reply:
It does not matter in this situation, this hashes later will get into map collection where service id is key.